bundle/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ ecbuild_bundle( PROJECT atlas GIT "https://github.com/ecmwf/atlas.git" TAG 0.31.
 # Core JEDI repositories
 # ----------------------
 ecbuild_bundle( PROJECT oops  GIT "https://github.com/jcsda/oops.git"  BRANCH develop UPDATE )
```

Comment:
Why not feature/Bapps for oops ?  
Saber PR #573 depends on oops PR #2213 but the test names change and the Randomized Variance section should disappear from references ...  
Shouldn't AQ be ready for dealing with both saber and oops PR at once ?

Reply:
Looks good to me.